scraper.rb

```diff
@@ -0,0 +1,20 @@
+require 'pry'
+require 'require_all'
+require 'scraped'
+require 'scraped_page_archive/open-uri'
+
+require_rel 'lib'
+
+# url = 'http://www.althingi.is/altext/cv/en/'
+urls = ['http://www.althingi.is/thingmenn/althingismenn/', # members
+        'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-sitja-a-althingi/', # alternates
+        'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-hafa-tekid-saeti/'] # substitues
```

Comment:
Rather than comments, perhaps make this a hash, and then iterate over the values?